src/tools/perf/perftest.c

```diff
@@ -721,9 +721,6 @@ static ucs_status_t parse_test_params(perftest_params_t *params, char opt,
         }
     case 'T':
         params->super.thread_count = atoi(opt_arg);
-        params->super.thread_mode  = (params->super.thread_count == 1) ?
```

Comment:
q: just wondering. does it mean that we will respect "-M" argument where thread mode is specified by the user?

Reply:
yes, thread mode is independent of # of threads, since each worker is on separate thread

Reply:
could there be any perf impact if we run MULTI if only 1 thread is used? 
do we want to allow this case?

Reply:
yes, there will be extra lock overhead, and this is something we want to test as well

Reply:
but isn't this considered a wrong usage? using Multi for 1 thread

Reply:
it's not optimal but i wouldn't say it's wrong

Reply:
ok